dotcom-rendering/src/web/lib/getZIndex.tsx

```diff
@@ -74,9 +74,9 @@ const indices = [
 // Implementation code - you don't need to change this to get a new index
 export type ZIndex = typeof indices[number];
 
-const decideIndex = (name: ZIndex): number | null => {
+const decideIndex = (name: ZIndex): number => {
 	const index = indices.indexOf(name);
-	if (index === -1) return null; // indexOf returns -1 if there is no match
+	if (index === -1) return -1; // indexOf returns -1 if there is no match
```

Comment:
This is in line with the usage, returning a value that will put the element below any other one.

This should never happen anyway, as the input is typed.